pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -428,37 +428,37 @@ type Framework interface {
 	// status other than Success.
 	RunPreFilterExtensionRemovePod(ctx context.Context, state *CycleState, podToSchedule *v1.Pod, podToAdd *v1.Pod, nodeInfo *NodeInfo) *Status
 
-	// RunPreScorePlugins runs the set of configured pre-score plugins. If any
+	// RunPreScorePlugins runs the set of configured PreScore plugins. If any
 	// of these plugins returns any status other than "Success", the given pod is rejected.
 	RunPreScorePlugins(ctx context.Context, state *CycleState, pod *v1.Pod, nodes []*v1.Node) *Status
 
-	// RunScorePlugins runs the set of configured scoring plugins. It returns a map that
+	// RunScorePlugins runs the set of configured Score plugins. It returns a map that
 	// stores for each scoring plugin name the corresponding NodeScoreList(s).
```

Comment:
```suggestion
	// stores for each Score plugin name and the corresponding NodeScoreList(s).
```

Reply:
btw. all those change should be applied to the IMPL fo those interface as well,  e.g.

https://github.com/kubernetes/kubernetes/blob/2d327ac4558d78c744004db178dacb80bd6e0b9e/pkg/scheduler/framework/runtime/framework.go#L601-L604